lumicks/pylake/kymotracker/tests/test_kymotrack.py

```diff
@@ -1505,3 +1516,28 @@ def test_photon_counts_api(blank_kymo):
         ).photon_counts,
         [1, 2, 5],
     )
+
+
+@pytest.mark.slow
+def test_integration_test_binding_times_with_minimum_time():
```

Comment:
name seems redundant. I'd suggest either
- `test_integration_binding_times_with_minimum_time`
- `binding_times_with_minimum_time_integration_test`

Reply:
It has to start with `test_` to be picked up by `pytest`, so I'd pick the former. 👍

Reply:
ah it's the file that can end with `test`, not the functions. that's what I was thinking about